Dockerfile

```diff
@@ -1,13 +1,26 @@
 FROM centos:7
+ARG JAVA_VERSION=1.8.0
 
-RUN yum -y install java-1.8.0-openjdk-devel && yum clean all
+RUN yum -y update \
+    && yum -y install java-${JAVA_VERSION}-openjdk-headless openssl \
+    && yum -y clean all
+
+# Set JAVA_HOME env var
 ENV JAVA_HOME /usr/lib/jvm/java
 
-ADD target/kafka-bridge.jar /
+# Add strimzi user with UID 1001
+# The user is in the group 0 to have access to the mounted volumes and storage
+RUN useradd -r -m -u 1001 -g 0 strimzi
```

Comment:
do we need that now? we don't have volumes and storage here.

Reply:
We should still not run the images under root since that is unsecure.